sphinx/themes/basic/static/basic.css_t

```diff
@@ -768,12 +768,16 @@ div.code-block-caption code {
     background-color: transparent;
 }
 
-table.highlighttable td.linenos,
-span.linenos,
-div.doctest > div.highlight span.gp {  /* gp: Generic.Prompt */
```

Comment:
Is it fine for you that I changed `div.doctest > div.highlight span.gp` to just `span.gp` ? 
Sometime, I have seen `$` sign outside a `doctest` `div` with the `gp` class, so I believe it would be better

Reply:
I don't know, the selector was added in #6727 by @tk0miya.

It seems to me that `gp` is a very un-specific name and might be used for something completely different by someone.

What about using `div.highlight span.gp`?
Would that cover the cases you are considering?

Reply:
Yep still work and I therefore push some changes. Thank you :-)

Reply:
If we remove `div.doctest`, select the prefixes in the code-blocks becomes not selectable (not copyable). Indeed, it will be helpful for almost of users. But I can't say ALL of the people want to do that. So this would not be changed, I think.